web/src/components/documents/document-card-view-item/document-card-view-item.module.scss

```diff
@@ -2,7 +2,7 @@
     position: relative;
```

Comment:
Do we really need to show this border by default? Should id be only appear when we drop something there?
<img width="1253" alt="image" src="https://github.com/epam/badgerdoc/assets/13063442/7c4e8a26-2039-469d-b4ce-353daf260d91">

Reply:
I think, we should show that it's a dnd zone and user could use this feature. In additional, we have different styles for dnd zone during overing files(blue borders and background)

Reply:
Okay,  let's keep it